UnityProject/Assets/Scripts/Core/Lighting/LightSource.cs

```diff
@@ -330,6 +331,12 @@ bool HasGlove()
 				{
 					foreach (var slot in handSlots)
 					{
+						if (interaction.PerformerMind.PossessingObject.TryGetComponent<Brain>(out var brain) &&
```

Comment:
playerHealth.brain.HasTelekinesis Might be better way of doing it idk tho

Reply:
both are the same really but i guess health is more reliable